src/main/java/gift/DTO/WishDTO.java

```diff
@@ -0,0 +1,26 @@
+package gift.DTO;
+
+import gift.model.Wish;
+
+public class WishDTO {
+    private Long wishId;
+    private Long productId;
+    private String productName;
+
+    public WishDTO(Wish wish) {
+        this.wishId = wish.getId();
+        this.productId = wish.getProduct().getId();
+        this.productName = wish.getProduct().getName();
+    }
+    public Long getWishId() {
```

Comment:
적절한 줄바꿈을 통해서 코드의 가독성을 높여주면 좋을 것 같아요. 

```suggestion
    }
    
    public Long getWishId() {
```